tests/testutils.sh

```diff
@@ -109,6 +109,9 @@ if [ -z "$PREFIX_CHANGED" ] && [ -n "$IDRIS2_PREFIX" ]; then
     export PREFIX_CHANGED=1
 fi
 
+# Set the most neutral locale for reproducibility
+LC_ALL=C.UTF-8
```

Comment:
```suggestion
export LC_ALL=C.UTF-8
```

Reply:
Well, I didn't do this because I was afraid of polluting the external context. But I tested that this does not which confuses me -- it means that I don't understand the difference. Fix without `export` works too.